.changeset/eleven-cougars-scream.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet": patch
+"@digdir/designsystemet-theme": patch
+---
+
+Add better support for `data-color` and `data-color-scheme` being defined on the same element
```

Comment:
Would this not classify as a fix?

Reply:
I noticed it does work in some browsers, but we can classify it as a fix

Reply:
Yeah, since we classified it as a bug, and I would expect this to work as this from the get go.